Classes/RollOff.lua

```diff
@@ -484,6 +484,16 @@ function RollOff:processRoll(message)
             return false;
         end)();
 
+        --- Check for stacked rolls
+        if (not RollType
+            and GL.Settings:get("StackedRoll.enabled")
+            and GL.StackedRoll:isStackedRoll(low, high)
+        ) then
+            RollType = {};
```

Comment:
Can be simplified

```
RollType = {
    [1] = GL.Settings:get("StackedRoll.identifier"),
    [4] = GL.Settings:get("StackedRoll.priority"),
};
```

Reply:
Oh, I didn't even know that was possible!